person.rb

```diff
@@ -2,27 +2,32 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
-  def can_use_service?
-    of_age? || parent_permission
+  def can_use_service
+    of_age || @parent_permission
   end
 
   def validate_name
     @name = @corrector.correct_name(@name)
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] For Your  `add_rental` method The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association, It may look something like this :point_down: 

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```